src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -68,22 +70,54 @@
 
     private boolean sessionEnabled = true;
 
-    @NotNull
```

Comment:
yes well, but the reference to the list itself should be not null (even if the list itself is empty)